### PR TITLE
Start league methods and helpers

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 class GameTeam
-  attr_reader :game_id, :team_id 
+  attr_reader :game_id, :team_id, :goals
```
Comment (denverdevelopments) [inline lib/game_team.rb:3]: Yes, need access to 'goals'

File: lib/teams_manager.rb [hunk 2]
```diff
@@ -16,5 +16,28 @@ def generate_list(data_path)
...
+    name_hash = get_names_hash
+    name_hash[team_id]
+  #alternate replacement without hashing.
+  #get_name(id)
+    # team = @teams.find do |team|
+    #   team.team_id == id
+    # end
+    # team.name
+  #end
+  end
```
Comment (Aphilosopher30) [inline lib/teams_manager.rb:41]: what bozo wrote this code?

File: lib/teams_manager.rb [hunk 3]
```diff
@@ -16,5 +16,28 @@ def generate_list(data_path)
...
+
+  def get_names_hash
+    team_names_hash = {}
+    @teams.each do |team|
+      team_names_hash[team.team_id] = team.team_name
+    end
+    team_names_hash
+  end
+
```
Comment (denverdevelopments) [inline lib/teams_manager.rb:30]: Possible Refactor, combine def get_names_hash & def get_team_name

**Review by Aphilosopher30 — APPROVED** [review]
good job over all.

**Review by denverdevelopments — APPROVED** [review]
Nice progress, good clean-up of 's' and spaces.

File: lib/teams_manager.rb (hunk 2 above)
Comment (Aphilosopher30) [reply lib/teams_manager.rb:41]: serious note: we need to decide wether we want to keep the hash approach, or not.

**Pull request merged**